DEVELOPMENT.md

```diff
@@ -117,7 +117,7 @@ make integration-test-down
 
 ## Run locally
 
-Running `make minikube-up NAME=provider` (set `provider` to the name of your chosen provider e.g. `NAME=vai`) will spin up a local K8s cluster with the operator deployed along with your chosen provider.
+Running `make minikube-up` will spin up a local K8s cluster with the operator deployed.
```

Comment:
Not a problem with this PR as such - but since you've made a change here I'll mention it

on line 83 we have the heading `## Running locally` and we also have this here `# Run locally`

We should combine these in some way